core/templates/dev/head/domain/learner_dashboard/LearnerDashboardIconsDirective.js

```diff
@@ -34,10 +34,15 @@ oppia.directive('learnerDashboardIcons', [
         'LearnerDashboardActivityIdsObjectFactory',
         'LearnerPlaylistService',
         function(
-            $scope, LearnerDashboardIdsBackendApiService,
-            LearnerDashboardActivityIdsObjectFactory,
-            LearnerPlaylistService) {
+          $scope, LearnerDashboardIdsBackendApiService,
+          LearnerDashboardActivityIdsObjectFactory,
+          LearnerPlaylistService) {
           $scope.activityIsCurrentlyHoveredOver = true;
+          $scope.playlistTooltipWatcher = false;
+
+          $scope.enablePlaylistTooltip = function (state) {
```

Comment:
I don't think you have followed the comments I made about this from the previous review. Please check that all comments from that review have been addressed. I'm going to stop reviewing here.

Reply:
Done Changes accordingly :+1:

Reply:
Done Changes accordingly :+1: